docs/using-airbyte/getting-started/oss-quickstart.md

```diff
@@ -220,6 +221,10 @@ Here's a list of common customizations.
 
 ## Migrating from Docker Compose (Optional)
 
+:::caution
+When deploying using Docker Compose, Airbyte didn't have specific requirements for resources like CPU and memory. However, this isn't the case for `abctl`. The minimum resources required to run Airbyte with `abctl` are 4 CPUs and 16 GB of RAM. Ensure your machine meets these requirements.
```

Comment:
What do you mean 4 cpus? If you mean the docker resources settings, clarify that.

Reply:
This is on standby, but thanks for the feedback!